pom.xml

```
@@ -33,18 +33,24 @@
 			<version>8.0.33</version>
 		</dependency>
 		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-data-jpa</artifactId>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+			<version>4.29.0</version>
```

```suggestion
			<version>${liquibase.version}</version>
```

spring boot parent provides this version